pkg/k8s/cnp.go

```diff
@@ -351,7 +351,7 @@ func (c *CNPStatusUpdateContext) updateViaKVStore(ctx context.Context, cnp *type
 	)
 
 	if err := <-kvstore.Client().Connected(ctx); err != nil {
-		return nil
+		return fmt.Errorf("kvstore is unavailable: %w", err)
```

Comment:
I think we are sticking to `fmt.Errorf()`. @tklauser has recently swapped instances of `errors.Wrap()` over to `fmt.Errorf()` again.

Reply:
FWIW, this was #12603. The idea was to use the functionality available in the Go standard library (since Go 1.13) rather than an external package for error wrapping.